packages/hdwallet-provider/typings/ganache-core/index.d.ts

```diff
@@ -0,0 +1,48 @@
+declare module "ganache-core" {
```

Comment:
Is there a good way to avoid the need to define this in every package? Like maybe ganache-core can just provide this? 

cc @davidmurdoch

Reply:
(I'm happy to figure this out later, but just wondering if it's easy to figure out now)

Reply:
ganachec-core hash shipped with typings since 2.7.0 https://github.com/trufflesuite/ganache-core/releases/tag/v2.7.0 via https://github.com/trufflesuite/ganache-core/commit/3988e13cc807bf707f573d877c7a10558c5f0095 Maybe we did it wrong?

Reply:
Took as an example from `interface-adapter`. Actually no need for it, deleted

Reply:
hot diggity! thanks for providing insight David